### PR TITLE
feat(settings/native): hide login/logout for 8x8.vc on profile screen

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

**Review by saghul — APPROVED** [review]

File: react/features/settings/components/native/ProfileView.tsx [hunk 1]
```diff
@@ -103,7 +105,9 @@ const ProfileView = ({ isInWelcomePage }: {
     useLayoutEffect(() => {
         navigation.setOptions({
             headerLeft,
-            headerRight: !isInWelcomePage && headerRight
+            headerRight: !isInWelcomePage
+                && !locationURL?.hostname?.includes('8x8.vc')
```
Comment (damencho) [inline react/features/settings/components/native/ProfileView.tsx:109]: This will detect not only jaas. Can we use the isVpass function?

Comment (damencho) [reply react/features/settings/components/native/ProfileView.tsx:109]: Like https://github.com/jitsi/jitsi-meet/commit/3811caa8a00114523c2f18df9c070522d1353d8c

Comment (saghul) [reply react/features/settings/components/native/ProfileView.tsx:109]: It's intentional, same should apply on 8x8.vc. Login on that domain won't work on mobile.

Comment (damencho) [reply react/features/settings/components/native/ProfileView.tsx:109]: Aham, as the commit says jaas. But if it will not work, ok.

Comment (saghul) [reply react/features/settings/components/native/ProfileView.tsx:109]: Right. Calin pl change the commit message when merging.

**Pull request merged**